stream-chat-android/src/main/res/layout/stream_message_list_view.xml

```diff
@@ -58,4 +58,38 @@
 
     </androidx.constraintlayout.widget.ConstraintLayout>
 
+    <FrameLayout
+        android:id="@+id/loadingViewContainer"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <ProgressBar
+            android:id="@+id/defaultProgressbar"
```

Comment:
It would be more consistent to name this `defaultLoadingView`, to match the pattern with the IDs of the empty views.